simphony/cuds/simulation.py

```diff
@@ -0,0 +1,42 @@
+"""Contains simulation controllers."""
+from . import CUDS
+from ..engine import get_wrapper
+
+
+class WrapperFactory(object):
+    """Creates wrappers."""
+    @staticmethod
+    def create(wrapper_name, cuds):
+        """Create a wrapper instance of the given name.
+
+        Parameters
+        ----------
+        cuds: CUDS
+            a cuds object containing simulation data
+        wrapper_name: str
+            A SimPhoNy wrapper name, should be of of registered engines.
+        """
+        wrapper = get_wrapper(wrapper_name)
+        if not wrapper:
+            raise Exception('Unknown wrapper: %s' % wrapper_name)
+
+        return wrapper(cuds=cuds)
+
+
+class Simulation(object):
+    """Represents a simulation using CUDS.
+
+    Parameters
+    ----------
+    cuds: CUDS
+        A cuds object which contains model information.
+    wrapper_name: str
+        Name of the wrapper to launch the simulation with.
+    """
+    def __init__(self, cuds, wrapper_name):
+        self._cuds = cuds
```

Comment:
This has been subject of lots of discussions here. Shall we have a cuds object that keeps track of its state changes or shall we have a separate bookkeeper object? Soon I will make changes to support multiple cuds. Consider this one as the initial state, i.e. time_step=0.